src/templates/credential.js

```diff
@@ -18,7 +18,7 @@ export default function TemplateCredential({
 
       ${subjectName ? `
         <p class="prettyVC-diploma-subtext">
-          has issued a <strong>${sanitize(title)}</strong> credential to
+          has issued a <strong>${sanitize(title)}</strong> to
```

Comment:
as a title, should it be in quotes?

Reply:
i dont think it looks right in quotes - i think were going to update this template soon anyway. it should read like: "xyz has issued a basic credential to zyx" or "xyz has issued a university degree to zyx"